.github/workflows/scripts/docker.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+HTTP_PROXY='http://10.24.221.149:911'
+HTTPS_PROXY='http://10.24.221.149:911'
+MODEL_CACHE_PATH_LOACL='/root/.cache/huggingface/hub'
+CODE_CHECKOUT_PATH_LOCAL='/root/llm-on-ray'
+
+build_and_prune() {
+    # Set TARGET and DF-SUFFIX using the passed in parameters
+    local TARGET=$TARGET
```

Comment:
Could you use $1, $2 to denote the arguments? it looks like this is not the standard way to pass in arguments?